### DISCUSSION TITLE
Thinking to use yantra into MelonRuntime/MelonJS

### DISCUSSION THREAD

**Discussion by scarletquasar** [yantrajs/yantra#46] [Q&A] [answered]
Do you people think it will fit better than Jint?
It have async/await and other interesting features.
Im thinking about use in: [MelonJS](https://github.com/MelonRuntime/MelonJS)

**Comment by ackava** [comment, marked as answer]
YantraJS already contains everything that MelonJS is trying to do, There is support for modules and support for node like inbuilt modules under development. Checkout CSX modules under https://github.com/yantrajs/yantra/wiki/JavaScript-Engine-Example, also in this folder, you will see couple of module examples. https://github.com/yantrajs/yantra/tree/main/modules/inbuilt

You can certainly use YantraJS in place of JInt.

There is support for generators and async/await. Have a look at feature list mentioned here.. https://github.com/yantrajs/yantra/wiki#features
